hearpreprocess/pipeline.py

```diff
@@ -665,9 +666,10 @@ def run(self):
             # duration
             # sample duration is specified in the task config.
             # The specified sample duration is in seconds
-            metadata = self.trim_event_metadata(
-                metadata, duration=self.task_config["sample_duration"]
-            )
+            if self.task_config["sample_duration"] is not None:
```

Comment:
Add a comment that if the sample_duration is None, we leave the length intact (but this is special for FSD50K which has a large discrepancy between shortest and longest lengths, and not the default).

Also add this documentation to schema.py